low-level/framework/utils/conf_utils.py

```diff
@@ -148,3 +152,8 @@
 
 SRVNODE = Conf.get(GLOBAL_CONF, f'{CLUSTER}>{SERVER_NODES}')[MACHINE_ID]
 ENCLOSURE = Conf.get(GLOBAL_CONF, f"{CLUSTER}>{SRVNODE}>{STORAGE}>{ENCLOSURE_ID}")
+
+SITE_ID = Conf.get(GLOBAL_CONF, f"{CLUSTER}>{SRVNODE}>{'site_id'}",'DC01')
```

Comment:
Keeping them on conf_utils doesnt makes sense with these changes, as these 4 entries no more relates to any util code. We can create a new file named global_config.py under sspl/low-level/framework/base, where all these cortx global config can be maintained / updated and made available for rest of the code, like sensors & actuators.

Reply:
So, In the global_config.py we will load the global_conf file. (#L149-151) and all the entites like MACHINE_ID, ENCLOSURE_ID, STORAGE etc will get imported from conf_utils.py.
Please confirm

Reply:
Yes